visualization/app/codeCharta/ui/codeMap/codeMap.arrow.service.ts

```diff
@@ -103,29 +128,40 @@ export class CodeMapArrowService implements BuildingHoveredSubscriber, BuildingU
 				arrowTargetNode.incomingEdgePoint
 			)
 
-			if (this.isHovered) {
-				this.hoveredMode(curve, arrowOriginNode, arrowTargetNode)
+			if (this.isClicked && this.isHovered) {
+				let bool = this.hoveredNode.path === arrowOriginNode.path || this.clickedNode.path === arrowOriginNode.path
```

Comment:
use `const` instead of let

Reply:
also rename bool to isOrigin or something along the lines